app/graphql/types/query_types/public_json_file_type.rb

```diff
@@ -3,6 +3,6 @@
 module Types
   class QueryTypes::PublicJsonFileType < Types::BaseObject
     field :name, String, null: false
-    field :content, String, null: false
+    field :content, GraphQL::Types::JSON, null: false
```

Comment:
are there more places you can think of, where we can update strings to real jsons?

Reply:
seemingly not